extensions/subscriptions/src/Listener/SendNotificationWhenReplyIsPosted.php

```diff
@@ -27,6 +27,12 @@ public function __construct(Queue $queue)
 
     public function handle(Posted $event)
     {
+        $discussion = $event->post->discussion;
+
+        /** @var \Psr\Log\LoggerInterface $log */
+        $log = resolve('log');
+        $log->info("running subscriptions send notification when reply is posted listener. last_post_number: $discussion->last_post_number");
+
```

Comment:
Hey @SychO9 just noticed this whilst looking at a seperate issue.

I'm not so sure we'd really want this output to the log, was this perhaps an accidental inclustion from some debugging or something?

Reply:
oh nice catch, that was for debugging in a different PR, it somehow slipped in while rebasing 🤔.